gotools.mk

```diff
@@ -0,0 +1,15 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+GOTOOLS = gocov gocov-xml goimports golint
+
+.PHONY: gotools
+gotools: $(patsubst %,build/tools/%, $(GOTOOLS))
+
+build/tools/%: tools/go.mod tools/tools.go
+	@mkdir -p $(@D)
+	@$(eval TOOL = ${subst build/tools/,,${@}})
+	@$(eval FQP = $(shell grep ${TOOL} tools/tools.go | cut -d " " -f2 | grep ${TOOL}\"$))
```

Comment:
@sykesm Do you have an issue with grepping the URL out of the tools.go file? Otherwise I need to maintain a map like we have in the Fabric repo of the URL's, and that's really what I was trying to avoid with grepping the tools out of the tools.go file above

Reply:
s/issue/concern/g - yes.